frontend/src/lang/en.ts

```diff
@@ -568,9 +568,9 @@ export const en = {
         dateOfBirth: "Date of birth",
         enterDOB: "Enter your date of birth",
         enterDOB2:
-          "Enter your date of birth to access your COVID-19 testing portal.",
+          "Enter <0>{{personName}}</0>'s date of birth to access your COVID-19 testing portal.",
         linkExpirationNotice:
-          "Note: this link will expire 5 days after the test result was recorded.",
+          "Note: this link will expire 10 days after the test result was recorded.",
```

Comment:
I know we're waiting on Spanish translations here, but can you change this variable from 5 -> 10 in the es.ts file before this change is merged?

Reply:
Excellent catch! 7629e99